### ISSUE TITLE
System Inspection: file location in language file

### ISSUE THREAD

**Issue by torvista** [zencart/zencart#6776]
I don't believe this is the appropriate location
https://github.com/zencart/zencart/blob/dd38315e34384fbea7711a796975e11d6125f626/zc_plugins/System%20Inspection/1.5.0/admin/includes/languages/english/extra_definitions/lang.menu_defines.php#L12

**Comment by drbyte** [comment]
You're right. It "works", but is not expected to be offered for translation, etc.
Will move it.
